src/module/data/item/effect.mjs

```diff
@@ -0,0 +1,89 @@
+import SFRPGItemBase from './base-item.mjs';
+
+const { fields } = foundry.data;
+
+export default class SFRPGItemEffect extends SFRPGItemBase {
+
+    static LOCALIZATION_PREFIXES = [
+        'SFRPG.Item.Base',
+        'SFRPG.Item.Effect'
+    ];
+
+    static defineSchema() {
+        const schema = super.defineSchema();
+
+        // merge schema with templates
+        foundry.utils.mergeObject(schema, {
+            ...SFRPGItemBase.itemDurationTemplate()
+        });
+
+        // Effect-specific properties
+        foundry.utils.mergeObject(schema, {
+            context: new fields.SchemaField({
+                origin: new fields.SchemaField({
+                    actorUuid: new fields.StringField({
```

Comment:
These two could use DocumentUUIDField, or ForeignDocumentField. I'm not sure which is to be used when